test/unit/test_iterative.py

```diff
@@ -0,0 +1,118 @@
+import gemmi
+import numpy as np
+import pandas as pd
+import pandas.testing as pdt
+import pytest
+import reciprocalspaceship as rs
+from skimage.data import binary_blobs
+from skimage.restoration import denoise_tv_chambolle
+
+from meteor import iterative
+from meteor.testing import assert_phases_allclose
+from meteor.tv import TvDenoiseResult
+from meteor.utils import compute_map_from_coefficients
+
+
+def map_norm(map1: gemmi.Ccp4Map, map2: gemmi.Ccp4Map) -> float:
```

Comment:
I wonder if we should add a test that looks at the negentropy metric for the map before/after itTV. What do you think?

Reply:
added negentropy check to test below